Cmdr/BuiltInCommands/Debug/versionServer.lua

```diff
@@ -0,0 +1,5 @@
+local version = "1.5.0"
+
+return function()
+	return ("Cmdr Version %s"):format(version)
+end
```

Comment:
This should be a client command. Use the `ClientRun` function in `version.lua` instead.